app/static/js/events/wizard/vue-app.js

```diff
@@ -61,6 +61,12 @@ var app = new Vue({
             var start = moment(this.event.start_time_date.trim() + " " + this.event.start_time_time.trim(), format);
             var end = moment(this.event.end_time_date.trim() + " " + this.event.end_time_time.trim(), format);
             return end.isBefore(start);
+        },
+        showTickets: function(){
+            if(this.event.tickets === null)
+                return 0;
+            else
+                return this.event.tickets.length > 0
```

Comment:
Like 79 returns 0 while this line returnss `true` or `false`. Can you try avoiding mixed return types, if possible?